edit/gatekeeper-constraint.vue

```diff
@@ -66,14 +66,17 @@ export default {
   data() {
     return {
       parametersYaml:           this.value?.spec?.parameters ? jsyaml.safeDump(this.value.spec.parameters) : '',
-      showParametersAsYaml:     !isSimpleKeyValue(this.value.spec.parameters),
+      showParametersAsYaml:     !isSimpleKeyValue(this.value?.spec?.parameters),
       enforcementActionOptions: Object.values(ENFORCEMENT_ACTION_VALUES),
       enforcementActionLabels:  Object.values(ENFORCEMENT_ACTION_VALUES).map(ucFirst),
       NAMESPACE_FILTERS
     };
   },
 
   computed: {
+    doneRoute() {
+      return 'c-cluster-gatekeeper-constraints';
```

Comment:
Just going to merge this because I'm sure it's better than before in general, but pretty sure this needs to handle which 'product' you're in to keep you in it (editing a constraint from explorer vs the gatekeeper product)